atom/browser/api/atom_api_web_contents.cc

```diff
@@ -382,6 +382,26 @@ void WebContents::HandleKeyboardEvent(
     // Escape exits tabbed fullscreen mode.
     ExitFullscreenModeForTab(source);
   } else if (type_ == BROWSER_WINDOW) {
+    // On OSX, if OS event is not existing (because it was forwarded with
+    // `sendInputEvent`) simulate basic operations: undo, redo, cut, copy,
+    // paste, select all.
+    #if defined(OS_MACOSX)
+      if (!event.os_event && (event.modifiers & blink::WebInputEvent::MetaKey)
+        && (event.type == blink::WebInputEvent::RawKeyDown)) {
+        switch (event.windowsKeyCode) {
+          case ui::VKEY_Z:
+            if (event.modifiers & blink::WebInputEvent::ShiftKey) Redo();
+            else
+              Undo();
+          break;
+          case ui::VKEY_X: Cut(); break;
+          case ui::VKEY_C: Copy(); break;
+          case ui::VKEY_V: Paste(); break;
+          case ui::VKEY_A: SelectAll(); break;
```

Comment:
Maybe it's possible to have custom behaviour for your chromium window's cmd+key combinations (though I think it's quite unlikely), but I couldn't find a solution that could trigger the these combinations with Cocoa. And without these it's quite frustrating to use the keyboard and that it's not consistent, because on Windows these keyboard events work nicely when sent with `sendInputEvent`.